### ISSUE TITLE
AWS_REGION/-r should not be required when using --profille

### ISSUE THREAD

**Issue by zdrummond** [apex/apex#413]
When using `apex --profile <profle_name>` I get `MissingRegion: could not find region configuration`, even though the env has been setup in my ~/.aws config.

If i leave off --profile, everything works and `apex` uses configured region

**Comment by tj** [comment]
Hmm and your ~/.aws/config has a `profile <profile_name>` section? When --profile is used we check that section, but maybe we should fall back on the default there as well. I'm not sure what other AWS tools do there semantically, but it makes sense to me

For ref: https://github.com/apex/apex/blob/master/utils/utils.go#L60-L83

**Comment by zdrummond** [comment]
Yup. I have a `config` file in in ~/.aws with that section name.

I noticed you log a failure with `fmt.Errorf("Could not find AWS region in %s", path)`. How can I see if that error is showing up?

**Comment by tj** [comment]
We just ignore it right now (https://github.com/apex/apex/blob/master/cmd/apex/root/root.go#L115-L120) since it's caught later, but we could use some more debug-level logging to help. That said it should be fine, not seeing any issues but I'll try and reproduce soon

**Comment by poscar** [comment]
👍 To falling back to default value!
